crates/storage/provider/src/bundle_state/state_reverts.rs

```diff
@@ -79,10 +91,17 @@ impl StateReverts {
             // Sort accounts by address.
             account_block_reverts.par_sort_by_key(|a| a.0);
             for (address, info) in account_block_reverts {
-                account_changeset_cursor.append_dup(
-                    block_number,
-                    AccountBeforeTx { address, info: info.map(into_reth_acc) },
-                )?;
+                if append_only {
+                    account_changeset_cursor.append_dup(
```

Comment:
can't select the lines, but the storage_changeset_cursor above requires this kind of change as well, right?

Reply:
@shekhirin ?